src/time-range-picker/types.ts

```diff
@@ -0,0 +1,29 @@
+export type TimeRangePickerFormat = "12hr" | "24hr";
+
+export interface TimeRangePickerProps {
+    startTime: string;
+    endTime: string;
+}
+
+export interface TimeRangePickerProps extends React.AriaAttributes {
```

Comment:
some thoughts on the typings
- no need to extend `AriaAttributes`
- `name`, `tabIndex`, and `onSelectionCancel` can be removed, I don't think they're relevant
- I also think `placeholder` and `defaultValue` can be removed for now, but if there's a use case for them, they should at least match the  typing of `value` since they should correspond to two inputs